src/src-admin/views/Login.js

```diff
@@ -43,7 +43,7 @@ const Login = (props) => {
     if (userInfo.email === 'athirahsn.hassan@gmail.com') {
       setUserInfo({...userInfo});
       sessionStorage.setItem("email",userInfo.email);
-      window.location.reload();
+      window.location.replace('http://localhost:3000/admin/home');
```

Comment:
A concern with referencing localhost:3000, when this is deployed, the url will be different, would it still work? To get the origin url (http://localhost:3000), you can use window.location.origin instead